pkg/server/plugin/datastore/sql/mysql.go

```diff
@@ -6,6 +6,7 @@ import (
 	"errors"
 	"io/ioutil"
 
+	"github.com/go-sql-driver/mysql"
 	mysqldriver "github.com/go-sql-driver/mysql"
```

Comment:
Hmm... is this new import line a mistake? Not sure if this named import was specifically to avoid the `mysql` struct name collision. Whether we rename the struct or leave the named import I don't have much preference.

I am surprised that the Go compiler allowed this.

Reply:
Oh gross. Yeah, the named import was because we had a struct named `mysql`, which I renamed because I had strong feels against it. I must have missed a mysqldriver import reference somewhere. I'll get rid of the named import.